src/components/BasicHead.vue

```diff
@@ -206,14 +196,21 @@ export default {
         this.$forceUpdate()
       }
     },
-    route$: {
+    '$route.name': {
       immediate: true,
       handler (val) {
         let name = this.$route.name
+        if(val) {
+          name = val
```

Comment:
won't line 202 and line 204 give the same value?

Reply:
True - I can never be so sure hence writing unnecessary code!

Reply:
Removed!